packages/features/bookings/Booker/Booker.tsx

```diff
@@ -82,6 +82,7 @@ const BookerComponent = ({
   isBookingDryRun: isBookingDryRunProp,
   renderCaptcha,
   hashedLink,
+  timeZonesFromProps,
```

Comment:
Can we call it `timeZones` instead? If they are in props type then they are coming from props so no need to add `FromProps` imo.